oeps/processes/oep-0055-project-maintainers.rst

```diff
@@ -0,0 +1,309 @@
+
+A new maintainer can be added in any of the following ways:
+
+* An existing maintainer can request that a new person become a
+  component maintainer either in conjunction with the existing
+  maintainer or replacing them.  If there is no objection from
+  existing maintainers, the role will be conferred.
+* In the case of vacancies, the nominations will be made to the
+  scrum-of-scrums to fill the vacancy.  If there is not objection from
+  existing maintainers, the role will be conferred.
+* If there consensus cannot be reached,
+* In the case of new repositories, there should be a designated
+  maintainer named before the repository is created.
+
+Removing a Maintainer
+---------------------
+From time-to-time it will be necessary to remove a component
+maintainer.  This might happen because:
+
+* A maintainer leaves the project or changes employment
+* A maintainer does not have the time or interest to continue in the
+  role
+* A maintainer is unreachable
+* A maintainer is not fulfilling the responsibilities of the role
+
+Trigger events for removing a maintainer would be:
+
+* A maintainer informs the project that the can no longer continue in the role
+* A maintainer has been unreachable for more than 30 days
+* A maintainer has violated the project `Code of Conduct`_
+* A maintainer hasn't met project quality standards consistently despite feedback
+  
+Jobs of the Maintainer
+----------------------
+
+Many of the day-to-day specifics of the maintainer role are outside of
+the scope of this document.  For example, here we will note that
+maintainers are responsible for ensuring that repositories meet
+project standards, but not the details of those standards.  Those
+standards will be codified in an OEP over time, but will initially be
+defined in the wiki page `requirements for public repositories`_ to
+allow rapid iteration.
+
+The key aspects of the maintainer role fall into the following
+categories
+
+* **Community Stewardship**: Maintainers are representatives of the
+  project in the community and are responsible for contributing to an
+  environment that is welcoming, vibrant, and growing.
+* **Project Management**: Maintainers are responsible for ensuring
+  that critical maintenance is done, but this doesn't mean doing it all
+  themselves. Effective delegation is an important maintainer skill.
+* **Quality Assurance**: Maintainers are responsible for the quality
+  of their components.  Quality will mean meeting defined project
+  standards.  It can also mean improving our standards or improving
+  our processes.
+* **Technical Vision**: Maintainers should have a technical vision for
+  their components. This would include key areas of debt that require
+  payoff, a new pattern that should be adopted to improve the
+  component design, a path to add new features user need, and more.
+* **Continuous Improvement**: Maintainers should be thinking
+  about continuously improving both software and processes.
```

Comment:
Thinking back to the GH Issues vs Jira tension, do you want to add something here? The maintainer role includes designating and monitoring a channel for feedback. It includes responding to that feedback.

Reply:
I was thinking that would go in the referenced wiki page, which would eventually become an OEP after we felt like it was adequately baked.